superset-frontend/src/views/CRUD/alert/types.ts

```diff
@@ -81,14 +82,15 @@ export type AlertObject = {
   sql?: string;
   timezone?: string;
   recipients?: Array<Recipient>;
-  report_format?: 'PNG' | 'CSV' | 'TEXT';
+  report_format?: 'PNG' | 'CSV' | 'TEXT' | string;
```

Comment:
We should make `report_format` an ENUM with the following fields

Reply:
Why are you adding `string` here? Are there other values other than PNG/CSV/TEXT that are valid here? If you add `string` it means that `report_format` can have any value.

+1 on using an enum like @hughhhh said.

Reply:
ok done